helios-integration-tests/src/test/resources/logback-test.xml

```diff
@@ -9,7 +9,6 @@
   </appender>
 
   <logger name="com.spotify.helios.client" level="info"/>
-  <logger name="com.spotify.hermes" level="info"/>
```

Comment:
intentional; why was this line ever here? No code in helios integration tests has this package name.

My guess is it was copy & pasted from somewhere else.